From affd5e819cb51c76bec8676064e1f99cae93d88a Mon Sep 17 00:00:00 2001
From: Masatake Iwasaki <iwasakims@apache.org>
Date: Thu, 13 Jul 2017 21:41:43 +0900
Subject: [PATCH 2557/2848] HADOOP-14646.
 FileContextMainOperationsBaseTest#testListStatusFilterWithSomeMatches
 never runs. Contributed by Andras Bokor.

(cherry picked from commit b61ab8573eb2f224481118004f620fe9f18db74b)
(cherry picked from commit 7c8d4a1b3fe07c26949a6b50e4b121e5cb3ac3b2)

Change-Id: I6f8b26772e47fe4bf4970fcdb68ac6d3f208129f
---
 .../fs/FileContextMainOperationsBaseTest.java      |    1 +
 1 file changed, 1 insertion(+)

diff --git a/hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/fs/FileContextMainOperationsBaseTest.java b/hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/fs/FileContextMainOperationsBaseTest.java
index 4d0ab7d..27bfec7 100644
--- a/hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/fs/FileContextMainOperationsBaseTest.java
+++ b/hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/fs/FileContextMainOperationsBaseTest.java
@@ -377,6 +377,7 @@ public void testListStatusFilterWithNoMatches() throws Exception {
     
   }
   
+  @Test
   public void testListStatusFilterWithSomeMatches() throws Exception {
     Path[] testDirs = {
         getTestRootPath(fc, TEST_DIR_AAA),
-- 
1.7.9.5

